src/ServiceInsight/ServiceControl/DefaultServiceControl.cs

```diff
@@ -236,23 +236,25 @@ IRestClient CreateClient(string baseUrl = null)
             return client;
         }
 
-        static RestRequestWithCache CreateMessagesRequest(string endpointName = null) => endpointName != null
-    ? new RestRequestWithCache(string.Format(EndpointMessagesEndpoint, endpointName), RestRequestWithCache.CacheStyle.IfNotModified)
-    : new RestRequestWithCache(MessagesEndpoint, RestRequestWithCache.CacheStyle.IfNotModified);
+        static RestRequestWithCache CreateMessagesRequest(string endpointName = null) =>
+            endpointName != null
+            ? new RestRequestWithCache(string.Format(EndpointMessagesEndpoint, endpointName), RestRequestWithCache.CacheStyle.IfNotModified)
+            : new RestRequestWithCache(MessagesEndpoint, RestRequestWithCache.CacheStyle.IfNotModified);
 
         PagedResult<T> GetPagedResult<T>(RestRequestWithCache request) where T : class, new()
         {
             var result = Execute<PagedResult<T>, List<T>>(request, response => new PagedResult<T>
             {
                 Result = response.Data,
-                TotalCount = int.Parse(response.Headers.First(x => x.Name == ServiceControlHeaders.TotalCount).Value.ToString())
+                TotalCount = int.Parse(
+                    response.Headers.FirstOrDefault(x => x.Name == ServiceControlHeaders.TotalCount)?.Value?.ToString() ?? "0")
```

Comment:
Is this line the only relevant change?

I.e. is the rest of this commit just whitespace fixes?